packages/ckeditor5-engine/tests/view/matcher.js

```diff
@@ -83,7 +92,141 @@ describe( 'Matcher', () => {
 			expect( matcher.match( el2 ) ).to.be.null;
 		} );
 
-		it( 'should match element attributes', () => {
+		it( 'should match all element attributes', () => {
+			const pattern = {
+				attributes: true
+			};
+			const matcher = new Matcher( pattern );
+			const el1 = new Element( document, 'p', { title: 'foobar' } );
+			const el2 = new Element( document, 'p', { title: ''	} );
```

Comment:
Maybe we could add another attribute to make sure that it's able to match more than one?